### PR TITLE
[Feature] build job topology and implement the epoch interface

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 42 / 89 (47.19%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|11|00.00%|[187, 188, 189, 190, 192, 193, 194, 196, 197, 198, 199]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[618]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[147, 160]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|30|03.33%|[106, 122, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 147, 148, 149, 150, 152, 153, 154, 156, 157]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|43|90.70%|[293, 306, 345, 413]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 41 / 46 (89.13%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[622]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[147]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|0|1|00.00%|[107]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|41|95.12%|[293, 346]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 86 / 106 (81.13%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[125, 126, 127, 128]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[913]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[622]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[147]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|0|1|00.00%|[107]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|41|95.12%|[293, 346]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|33|33|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 85 / 105 (80.95%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[125, 126, 127, 128]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[913]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[622]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[147]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|0|1|00.00%|[107]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|41|95.12%|[293, 346]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 85 / 105 (80.95%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[136, 137, 138, 139]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[913]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[157]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|0|1|00.00%|[111]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|41|95.12%|[293, 347]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|2|2|100.00%|[]|

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 90 / 111 (81.08%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[136, 137, 138, 139]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[913]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|1|00.00%|[157]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|42|92.86%|[294, 310, 349]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 90 / 113 (79.65%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[136, 137, 138, 139]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[913]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|43|90.70%|[294, 307, 311, 350]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 90 / 113 (79.65%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[136, 137, 138, 139]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|39|43|90.70%|[294, 307, 311, 350]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Review by Seaven — DISMISSED** [review]

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -675,51 +685,54 @@ Status PInternalServiceImplBase<T>::_submit_mv_maintenance_task(brpc::Controller
...
-        }
+        RETURN_IF_ERROR(_mv_start_epoch(query_ctx, t_request));
         break;
     }
     case MVTaskType::COMMIT_EPOCH: {
+        RETURN_IF_ERROR(_mv_commit_epoch(query_ctx, t_request));
+
+        auto& commit_epoch = t_request.commit_epoch;
+        auto& version_info = commit_epoch.partition_version_infos;
+        std::stringstream version_str;
```
[the remark this answers is not in the document]
Comment (mofeiatwork) [reply be/src/service/internal_service.cpp:722]: fixed

File: be/src/column/stream_chunk.cpp [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#include "gen_cpp/MVMaintenance_types.h"
+
+namespace starrocks {
+
+EpochInfo EpochInfo::from_start_epoch_task(const TMVStartEpochTask& start_epoch) {
+    EpochInfo res;
+    res.epoch_id = start_epoch.epoch.epoch_id;
+    res.txn_id = start_epoch.epoch.txn_id;
+    res.max_exec_millis = start_epoch.max_exec_millis;
+    res.max_scan_rows = start_epoch.max_scan_rows;
```
[the remark this answers is not in the document]
Comment (mofeiatwork) [reply be/src/column/stream_chunk.cpp:26]: fixed

File: be/src/agent/publish_version.h [hunk 3]
```diff
@@ -25,7 +26,7 @@ namespace starrocks {
 class ThreadPoolToken;
 class DataDir;
 
-void run_publish_version_task(ThreadPoolToken* token, const PublishVersionAgentTaskRequest& publish_version_task,
+void run_publish_version_task(ThreadPoolToken* token, const TPublishVersionRequest& publish_version_task,
```
[the remark this answers is not in the document]
Comment (mofeiatwork) [reply be/src/agent/publish_version.h:29]: No. The implement is same as before, but only change the parameter

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 92 / 124 (74.19%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|4|00.00%|[136, 137, 138, 139]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|8|00.00%|[511, 515, 519, 520, 523, 527, 531, 535]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|41|54|75.93%|[242, 243, 244, 245, 247, 248, 249, 250, 251, 309, 322, 326, 365]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 92 / 127 (72.44%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|5|00.00%|[136, 137, 138, 139, 140]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|10|00.00%|[511, 515, 519, 520, 523, 524, 527, 531, 535, 539]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|41|54|75.93%|[242, 243, 244, 245, 247, 248, 249, 250, 251, 309, 322, 326, 365]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 92 / 127 (72.44%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|5|00.00%|[136, 137, 138, 139, 140]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|10|00.00%|[511, 515, 519, 520, 523, 524, 527, 531, 535, 539]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|41|54|75.93%|[242, 243, 244, 245, 247, 248, 249, 250, 251, 309, 322, 326, 365]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 101 / 140 (72.14%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|7|00.00%|[99, 100, 134, 135, 136, 137, 138]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|10|00.00%|[511, 515, 519, 520, 523, 524, 527, 531, 535, 539]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/qe/QeProcessorImpl.java|6|8|75.00%|[153, 154]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|44|57|77.19%|[243, 244, 245, 246, 248, 249, 250, 251, 252, 312, 325, 329, 370]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 102 / 152 (67.11%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVManager.java|0|11|00.00%|[84, 101, 103, 104, 105, 106, 140, 141, 142, 143, 144]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|0|1|00.00%|[971]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|0|10|00.00%|[511, 515, 519, 520, 523, 524, 527, 531, 535, 539]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|0|1|00.00%|[623]|
|:large_blue_circle:|com/starrocks/scheduler/mv/TxnBasedEpochCoordinator.java|0|2|00.00%|[157, 172]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceTask.java|1|2|50.00%|[110]|
|:large_blue_circle:|com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java|3|5|60.00%|[322, 323]|
|:large_blue_circle:|com/starrocks/scheduler/mv/MVMaintenanceJob.java|45|65|69.23%|[127, 128, 129, 130, 131, 132, 133, 257, 258, 259, 260, 262, 263, 264, 265, 266, 326, 339, 343, 384]|
|:large_blue_circle:|com/starrocks/qe/QeProcessorImpl.java|6|8|75.00%|[153, 154]|
|:large_blue_circle:|com/starrocks/sql/plan/PlanFragmentBuilder.java|32|32|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|9|9|100.00%|[]|
|:large_blue_circle:|com/starrocks/system/ComputeNode.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/qe/CoordinatorPreprocessor.java|5|5|100.00%|[]|

**Review by kangkaisen — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**